packages/compartment-mapper/src/link.js

```diff
@@ -14,6 +14,11 @@
 
 import { resolve } from './node-module-specifier.js';
 import { parseExtension } from './extension.js';
+import {
+  getAllowedGlobals,
+  gatekeepModuleAccess,
+  attenuateModuleHook,
+} from './policy.js';
```

Comment:
It is possible that I did not identify all places where these functions need to be called in this file.   
I plan to address that with more tests.   
Hints are welcome though.